internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -1674,28 +1674,7 @@ func resourceLinuxVirtualMachineDelete(d *pluginsdk.ResourceData, meta interface
 		return fmt.Errorf("retrieving Linux %s: %+v", id, err)
 	}
 
-	if !meta.(*clients.Client).Features.VirtualMachine.SkipShutdownAndForceDelete {
-		// If the VM was in a Failed state we can skip powering off, since that'll fail
-		if model := existing.Model; model != nil && model.Properties != nil && model.Properties.ProvisioningState != nil {
-			if strings.EqualFold(*existing.Model.Properties.ProvisioningState, "failed") {
-				log.Printf("[DEBUG] Powering Off Linux Virtual Machine was skipped because the VM was in %q state %s", *model.Properties.ProvisioningState, id)
-			} else {
-				// ISSUE: 4920
-				// shutting down the Virtual Machine prior to removing it means users are no longer charged for some Azure resources
-				// thus this can be a large cost-saving when deleting larger instances
-				// https://docs.microsoft.com/en-us/azure/virtual-machines/states-lifecycle
-				log.Printf("[DEBUG] Powering Off Linux %s", id)
-				skipShutdown := !meta.(*clients.Client).Features.VirtualMachine.GracefulShutdown
-				options := virtualmachines.PowerOffOperationOptions{
-					SkipShutdown: pointer.To(skipShutdown),
-				}
-				if err := client.PowerOffThenPoll(ctx, *id, options); err != nil {
-					return fmt.Errorf("powering off Linux %s: %+v", id, err)
-				}
-				log.Printf("[DEBUG] Powered Off Linux %s", id)
-			}
-		}
-	}
+	// Due to a breaking change in the API the power down (e.g., 'graceful_shutdown') feature flag is no longer supported...
```

Comment:
Can remove this comment, it will have no meaning post merge.
```suggestion
```

Reply:
Fixed.